tests/phpunit/unit-tests/class-llms-test-assets.php

```diff
@@ -723,64 +723,6 @@ public function test_register_script_undefined() {
 
 	}
 
-	/**
-	 * Test register_script() with translations
-	 *
-	 * @since 5.5.0
-	 *
-	 * @return void
-	 */
-	public function test_register_script_with_translations() {
```

Comment:
From what I can see this test isn't testing what we want it to, since the translations are working as expected.